### PR TITLE
Minor edits to SC CLI, YAML config

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/cli/sauce-connect-proxy.md [hunk 1]
```diff
@@ -44,23 +44,23 @@ __Shorthand__: `-k`
...
 
 
 ---
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
-__Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Description__: Sets your Sauce Labs regional data center such as `us-west` and `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
+__Default__: If you don't specify a data center, it will default to `us-west`.<br/>
```
Comment (sweeneyskirt-sl) [inline docs/dev/cli/sauce-connect-proxy.md:59]:
```suggestion
__Description__: Sets your Sauce Labs regional data center, such as `us-west` or `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
__Default__: Defaults to `us-west` when not specified.<br/>
```

File: docs/dev/cli/sauce-connect-proxy.md [hunk 2]
```diff
@@ -311,15 +312,14 @@ __Shorthand__: n/a
 ### `--ocsp`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: OCSP verification mode. Options are: strict, log-only, and disable. The default is log-only.
+__Description__: Sets OCSP verification mode. Options are: `strict`, `log-only`, and `disable`.<br/>
+__Default__: log-only<br/>
```
Comment (sweeneyskirt-sl) [inline docs/dev/cli/sauce-connect-proxy.md:316]:
```suggestion
__Default__: Defaults to `log-only` when not specified.<br/>
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
Hey @kimsauce -- Make a couple observations here. I am concerned that some of your edits overlap with edits that @waggledans made in his environment variables PR and we may end up with merge conflicts when one of them gets merged, so let's make sure we keep an eye on that and make sure the changes are well-coordinated. --Nancy

File: docs/dev/cli/sauce-connect-proxy.md [hunk 3]
```diff
@@ -44,23 +44,23 @@ __Shorthand__: `-k`
...
 __Shorthand__: `-c`
 
 
 ---
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
-__Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Description__: Sets your Sauce Labs regional data center such as `us-west` and `eu-central`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
```
Comment (waggledans) [inline docs/dev/cli/sauce-connect-proxy.md:58]: maybe "such as `us-west` or `eu-central`"

File: docs/dev/cli/sauce-connect-proxy.md [hunk 4]
```diff
@@ -130,29 +130,29 @@ __Shorthand__: `-F`
...
-__Description__: Sets domain(s) that need to be sent through the Sauce Connect Proxy tunnel. This is the inverse of `--direct-domains`.  When adding multiple domains, [format as a comma-separated list](#formatting-domains-in-the-command-line). Be sure to format your domains as a comma-separated list (see [Formatting Domains guidelines](#formatting-domains-in-the-command-line)).<br/>
+__Description__: Sets domain(s) that need to be sent through the Sauce Connect Proxy tunnel. This is the inverse of `--direct-domains`.  When adding multiple domains, [format as a comma-separated list](#formatting-domains-in-the-command-line).<br/>
 __Shorthand__: `-t`
 
 
 ---
 ### `--rest-url`
 <p><small>| OPTIONAL | STRING |</small></p>
 
+__Description__: Sets your [Sauce Labs regional data center REST API URL](#data-center-endpoints) (e.g., EU-Central, US-West).<br/>
```
[the remark this answers is not in the document]
Comment (waggledans) [reply docs/dev/cli/sauce-connect-proxy.md:141]: I think should remove `(e.g., EU-Central, US-West).`

**Pull request closed without merging**